website/docs/d/app_service_plan.html.markdown

```diff
@@ -44,6 +44,8 @@ output "app_service_plan_id" {
 
 * `maximum_number_of_workers` - The maximum number of workers supported with the App Service Plan's sku.
 
+* `maximum_elastic_worker_count` - The maximum number of total workers allowed for this ElasticScaleEnabled App Service Plan.
```

Comment:
Could we put this before `maximum_number_of_workers`?

Reply:
I am happy to do this - I am not sure why though but 🤷‍♂